tests/js/chartAuthoringTests.js

```diff
@@ -78,15 +118,25 @@ https://github.com/floe/universal/LICENSE.txt
         fluid.each(that.templateLoader.resources, function (resource, resourceName) {
             jqUnit.assertValue("The resource text for " + resourceName + " should have been fetched", resource.resourceText);
         });
-        jqUnit.assertEquals("The dataEntryPanel has not been rendered", "", that.container.html());
+        jqUnit.assertUndefined("The dataEntryPanel has not been instantiated", that.dataEntryPanel);
+        jqUnit.assertUndefined("The pieChart has not been instantiated", that.pieChart);
     };
 
-    floe.tests.chartAuthoringTester.verifyPanel = function (that, dataEntryPanel) {
-        fluid.each(that.templateLoader.resources, function (resource, resourceName) {
-            jqUnit.assertDeepEq("Templates have been passed into the dataEntryPanel sub-component", resource.resourceText,
-                dataEntryPanel.options.resources[resourceName === "dataEntryPanel" ? "template": resourceName].resourceText);
-        });
-        jqUnit.assertNotEquals("The dataEntryPanel has been rendered", "", dataEntryPanel.container.html());
+    floe.tests.chartAuthoringTester.verifyTool = function (that) {
```

Comment:
Can you also verify the new subcomponent ```chartAuthoringInterface```, such as its instantiation and its template being passed correctly? Thanks.